alchemy/src/alchemy.ts

```diff
@@ -314,7 +314,7 @@ async function _alchemy(
   ].join("\n");
 }
 
-export type Phase = "up" | "destroy" | "read";
+export type Phase = "up" | "destroy" | "read" | "dev";
```

Comment:
I am not sure if dev should be a phase. I think it's a qualifier. I can run in up+dev or destroy+dev

Some resources will ignore the qualifier and just operate normally 

Others can override behavior when mode is dev.

Reply:
Updated this per our conversation, LMK if I got it right!